middleware/http/oauth2clientcredentials/oauth2clientcredentials_middleware.go

```diff
@@ -142,6 +157,14 @@ func (m *Middleware) getNativeMetadata(metadata middleware.Metadata) (*oAuth2Cli
 	m.checkMetadataValueExists(&errorString, &middlewareMetadata.Scopes, "scopes")
 	m.checkMetadataValueExists(&errorString, &middlewareMetadata.TokenURL, "tokenURL")
 
+	if middlewareMetadata.PathFilter != "" {
+		rx, err := regexp.Compile(middlewareMetadata.PathFilter)
+		if err != nil {
+			errorString += fmt.Sprintf("Parameter 'pathFilter' is not a valid regex: %s. ", err)
```

Comment:
```suggestion
			errorString += "Parameter 'pathFilter' is not a valid regex: " + err + ". "
```
its more performant this way :)